tests/conftest.py

```diff
@@ -546,9 +546,7 @@ def full_record(users):
             ],
             "references": [
                 {
-                    "reference": "Nielsen et al,..",
-                    "identifier": "0000 0001 1456 7559",
-                    "scheme": "isni",
+                    "reference": "0000 0001 1456 7559",
```

Comment:
Updated test records, references are stored as 
```
    "references": [
      {
      "reference": "0000 0001 1456 7559"
      }
    ]
```
instead of
```
            "references": [
                {
                    "identifier": "0000 0001 1456 7559",
                    "reference": "Nielsen et al,..",
                    "scheme": "isni",
                },
            ],
```
<img width="1110" alt="Screenshot 2024-04-04 at 10 57 42" src="https://github.com/inveniosoftware/invenio-rdm-records/assets/51617644/c5483673-c842-4534-bfff-feaf48efd8c3">